frontend/src/stores/media/index.ts

```diff
@@ -499,15 +499,17 @@ export const useMediaStore = defineStore("media", {
         })
         return mediaCount
       } catch (error: unknown) {
-        const errorData = parseFetchingError(error, mediaType, "search", {
-          searchTerm: queryParams.q ?? "",
-        })
+        const errorData = this.$nuxt.$processFetchingError(
+          error,
+          mediaType,
+          "search",
+          {
+            searchTerm: queryParams.q ?? "",
+          }
```

Comment:
```suggestion
          { searchTerm: queryParams.q ?? "" }
```
Nit: these seem more readable when on a single line